TODO

```diff
@@ -24,6 +24,10 @@ Janitorial Clean-ups:
 
 Features:
 
+* log.c: change handling if an error code == 0 is passed: don't respect pre-set
+  errno in that case, like we currently do, but print error string of
+  "Success".
```

Comment:
I wanted to say "But this is already covered by b29f6480eca0550ba65d30fbece8dd4d4bfe666d", but I see I missed a few spots. I'll submit a PR. I think you can drop this commit though, it should be fixed soon either way.